<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonCommitQ">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>提交查询</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSQL">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidgetDB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>数据库总览</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_U">
    <property name="title">
     <string>用户(&amp;U)</string>
    </property>
    <addaction name="action_L"/>
    <addaction name="action_R"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>数据库操作(&amp;D)</string>
    </property>
    <addaction name="action_C"/>
    <addaction name="action_CreateTable"/>
    <addaction name="action_E"/>
   </widget>
   <addaction name="menu_U"/>
   <addaction name="menu_R"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_L">
   <property name="text">
    <string>登录(&amp;L)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>注册(&amp;R)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>创建/选择数据库(&amp;C)</string>
   </property>
  </action>
  <action name="action_CreateTable">
   <property name="text">
    <string>创建表(&amp;T)</string>
   </property>
  </action>
  <action name="action_E">
   <property name="text">
    <string>编辑表(&amp;E)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>修改权限(&amp;Q)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
